src/pages/components/footerSection/FooterContent.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const FooterContent = () => {
+  return (
+    <div className="bg-main-backgound">
```

Comment:
remove the background, instead make use of this https://github.com/framgia/sph_ui-ux-swipet/pull/6/files#r861418188